tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -362,108 +363,31 @@ public function theGroupShouldExist(string $groupname):bool {
 	}
 
 	/**
-	 *
-	 * @param string $groupname
-	 *
-	 * @return boolean
-	 * @throws Exception
-	 */
-	public function theGroupShouldBeAbleToBeDeleted(string $groupname):bool {
-		if (\array_key_exists($groupname, $this->createdGroups)) {
-			return $this->createdGroups[$groupname]['possibleToDelete'] ?? true;
-		}
-
-		if (\array_key_exists($groupname, $this->createdRemoteGroups)) {
-			return $this->createdRemoteGroups[$groupname]['possibleToDelete'] ?? true;
-		}
-
-		throw new Exception(
-			__METHOD__
-			. " group '$groupname' was not created by this test run"
-		);
-	}
-
-	/**
-	 * @When /^the administrator creates user "([^"]*)" using the provisioning API$/
-	 *
-	 * @param string|null $user
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function adminCreatesUserUsingTheProvisioningApi(?string $user):void {
-		$this->createUser(
-			$user,
-			null,
-			null,
-			null,
-			true,
-			'api'
-		);
-		$this->pushToLastStatusCodesArrays();
-	}
-
-	/**
-	 * @Given /^user "([^"]*)" has been created with default attributes in the database user backend$/
-	 *
-	 * @param string|null $user
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function userHasBeenCreatedOnDatabaseBackend(?string $user):void {
-		$this->adminCreatesUserUsingTheProvisioningApi($user);
-		$this->userShouldExist($user);
-	}
-
-	/**
-	 * @Given /^user "([^"]*)" has been created with default attributes and (tiny|small|large)\s?skeleton files$/
+	 * @Given user :user has been created with default attributes and without skeleton files
```

Comment:
I think we can remove the skeleton part as well but can be a part of some other PR
```suggestion
	 * @Given user :user has been created with default attributes
```